applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -42,23 +42,39 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
     using Element::Element;
 
     /// Constructor using an array of nodes
-    UPwBaseElement(IndexType NewId, const NodesArrayType& ThisNodes, std::unique_ptr<StressStatePolicy> pStressStatePolicy)
-        : Element(NewId, ThisNodes), mpStressStatePolicy{std::move(pStressStatePolicy)}
+    UPwBaseElement(IndexType                                       NewId,
+                   const NodesArrayType&                           ThisNodes,
+                   std::unique_ptr<StressStatePolicy>              pStressStatePolicy,
+                   std::unique_ptr<IntegrationCoefficientModifier> pCoefficientModifier = nullptr)
+        : Element(NewId, ThisNodes),
+          mpStressStatePolicy{std::move(pStressStatePolicy)},
+          mpIntegrationCoefficientsCalculator{
+              std::make_unique<IntegrationCoefficientsCalculator>(std::move(pCoefficientModifier))}
```

Comment:
When this data member is no longer a `std::unique_ptr` instance, initializing it will simplify to:
```suggestion
          mpIntegrationCoefficientsCalculator{std::move(pCoefficientModifier)}
```

Reply:
done